src/games/strategy/engine/ClientFileSystemHelper.java

```diff
@@ -135,7 +132,7 @@ public static File getUserRootFolder() {
     final File userHome = new File(System.getProperties().getProperty("user.home"));
     // the default
     File rootDir;
-    if (GameRunner.isMac()) {
+    if (System.getProperties().getProperty("os.name").toLowerCase().contains("mac")) {
```

Comment:
That is the problem with ClientContext, since we are not using an IOC framework, whatever creates that context can't also be in turned called by client context.

Let's see if we can avoid "System.getProperties().getProperty("os.name").toLowerCase().contains("mac")". I do still think you could skip the check and just do:
```
if(  new File(new File(userHome, "Documents"), "triplea").exists() ) {
  // use it
} else { 
  rootDir = new File(userHome, "triplea");
 }
```